zebra-chain/Cargo.toml

```diff
@@ -27,7 +27,7 @@ thiserror = "1"
 x25519-dalek = { version = "0.6", features = ["serde"] }
 serde-big-array = "0.3.0"
 # ZF deps
-ed25519-zebra = "0.4"
+ed25519-zebra = "2.0"
```

Comment:
Based on the docs and ZIP-215, we need to use:
* 1.0 and **no batch verification** for pre-Canopy blocks, and
* 2.0 with optional batch verification for post-Canopy blocks:
https://github.com/ZcashFoundation/ed25519-zebra#zip-215-and-changes-to-zcash-flavored-ed25519

This is a consensus-critical change.